### ISSUE TITLE
WCAG issue (2.4.4 Link Purpose (In Context))

### ISSUE THREAD

**Issue by sorensensig** [digdir/baksia#919]
List of issues related to 2.4.4 Non-text Content found during WCAG walkthrough of Baksia:

```[tasklist]
### 404 Link issues
- [ ] 404 error on Backlog and Sprint links due to locked board on the [Styringssystem team page](https://baksia.digdir.no/teams/styringssystem/)
- [ ] 404 error on Backlog link due to locked board on the [Altinn-plattform team page](https://baksia.digdir.no/teams/altinn-platform/)
- [ ] 404 error on Roadmap link due to locked board on the [Apps team page](https://baksia.digdir.no/teams/apps/)
- [ ] 404 error on Roadmap, Sprint and Backlog due to locked board on the [Core team page](https://baksia.digdir.no/teams/core/)
- [ ] 404 error on Status due to locked board on the [Portfolio team page](https://baksia.digdir.no/teams/portfolio/)
- [ ] 404 error on the Rollebeskrivelser link under Marked og tjenesteoppfølgning on the [Seksjoner page](https://baksia.digdir.no/brukeropplevelse-og-datadeling/seksjoner/)
- [ ] 404 error on the Websak+ link on the [verktøy page ](https://baksia.digdir.no/brukeropplevelse-og-datadeling/verktoy-og-infokanal/)
```

```[tasklist]
### Incorrect links
- [ ] Incorrect link to Roadmap on the [Styringssystem team page ](https://baksia.digdir.no/teams/styringssystem/)
- [ ] Incorrect link to Roadmap, Sprint and Backlog on the [Altinn II team page ](https://baksia.digdir.no/teams/altinn-2/)
- [ ] Incorrect link to Roadmap, Sprint and Backlog on the [App migration team page](https://baksia.digdir.no/teams/app-migration/)
- [ ] Incorrect link to Roadmap, Sprint and Backlog on the [Digital gravferd team page](https://baksia.digdir.no/teams/digital-gravferd/)
- [ ] Incorrect link to Roadmap, Sprint and Backlog on the [Digitalt dødsbo team page](https://baksia.digdir.no/teams/digitalt-dodsbo/)
- [ ] Incorrect link to Roadmap, Sprint and Backlog on the [Melding & Formidling team page](https://baksia.digdir.no/teams/formidling/)
- [ ] Incorrect link to to Backlog on the [Studio team page](https://baksia.digdir.no/teams/studio/)
```

Hitting a log in wall is not a problem if the user is informed about it prior to clicking the link and the supporting aria-label let's users in need of accessibility measures know the same information prior to clicking the link.
```[tasklist]
### Log-in walls
- [ ] Log-in wall and incorrect icon for both links (not slack) on the [Marked team page](https://baksia.digdir.no/teams/marked/)
- [ ] Log-in wall and incorrect icon for all three links (not slack) on the [OKR team page](https://baksia.digdir.no/teams/okr/)
- [ ] Log-in wall for the role descriptions on the [Roller og ressurser page](https://baksia.digdir.no/produktutviklingsmodell/roller/)
- [ ] Log-in wall on all links on the [operations page](https://baksia.digdir.no/produktutviklingsmodell/prinsipper/operations/)
- [ ] Log-in wall for the following links: Styringskalender Digdir, Statistikk for ABS, Servicerapporter, Virksomhetsplanlegging and Trusselvurderinger i Digdir on the [Intern Rapportering page](https://baksia.digdir.no/brukeropplevelse-og-datadeling/rapportering/)
- [ ] Log-in wall for all but one link on the [Onboarding page](https://baksia.digdir.no/brukeropplevelse-og-datadeling/onboarding/)
- [ ] Log-in wall for several links on the [verktøy page](https://baksia.digdir.no/brukeropplevelse-og-datadeling/verktoy-og-infokanal/)
```

```[tasklist]
### Other issues
- [ ] Potential duplicate links (datalandsbyen & datajegeren) on the [product group datadeling page](https://baksia.digdir.no/produktgrupper/datadeling/)
```

**Comment by altinnadmin** [comment]
Had a look, changed to v2
